src/main/cli/cli.c

```diff
@@ -4162,6 +4162,8 @@ STATIC_UNIT_TESTED void cliSet(char *cmdline)
                         strncpy((char *)cliGetValuePointer(val), valPtr, len);
                     }
                     valueChanged = true;
+                } else {
+                    cliPrintErrorLinef("STRING MUST BE 1-%d CHARACTERS", max);
```

Comment:
How 'bout `STRING MUST BE 1-%d CHARACTERS OR '-' FOR EMPTY`?

Reply:
Ok - I'll update

Reply:
Updated the error message to:
```
# set name = jsdfldjsflsjdflkjsdlfkjsdlfkjsd
###ERROR: STRING MUST BE 1-16 CHARACTERS OR '-' FOR EMPTY###
###ERROR: INVALID VALUE###
```